### ISSUE TITLE
Notebook on 3D data and on images from multiple imaging cycles

### ISSUE THREAD

**Issue by LucaMarconato** [scverse/spatialdata-notebooks#23]
Two related concepts that would be nice to show, and show in the same notebook (from a discussion with @lopollar and @melonora).

Sometimes the user has a large 3D image and wants to operate on a 2D slice, and sometime the users have many two images and want to stack them together in a 3D image, or in a 2D multichannel image (the stacking is not just a "`numpy` array stacking", but it could involve rotations, etc and requires re-rasterization and cropping).

After the new transformations refactoring and the merge of the `stitch` notebook this will be supported and it would be nice to show this to the user, together with a napari viz of the 2D layers embedded in the 3D space.

**Comment by yuling999666** [comment]
Hi @LucaMarconato, I have questions about alignment and stacking. I have noticed that spatialData used H&E image and landmarks to do alignment instead of using gene expression data and spatial coordinates? The alignment can just be generated by manually select landmarks or set specific parameters of transformation, rotations..etc. Is that correct? I think the alignment cannot be generated automatically. Thank you.

**Comment by LucaMarconato** [comment]
Exactly, we do not have a method for automatic alignment; rather once the alignment transformation is found (for instance by manually annotated landmarks), the `spatialdata` APIs allow conveniently operate with those transformations (e.g. visualize transformed data, transfer the alignment to other elements, etc).
